cfme/containers/image.py

```diff
@@ -39,6 +42,22 @@ def get_detail(self, *ident):
         navigate_to(self, 'Details')
         return InfoBlock.text(*ident)
 
+    def perform_smartstate_analysis(self, wait_for_finish=False):
+        """Performing SmartState Analysis on this Image
+        """
+        navigate_to(self, 'Details')
```

Comment:
Because this navigation destination isn't paged your test is failing to find the correct image for analysis.